spec/2e_extensions-mechanism.adoc

```diff
@@ -49,8 +50,8 @@ See <<gpkg_extensions_sql>>.
 [caption=""]
 .Requirement 59
 ====
-Every extension of a GeoPackage SHALL be registered in a corresponding row in the gpkg_extensions table.
-The absence of a gpkg_extensions table or the absence of rows in the gpkg_extensions table SHALL both indicate the absence of extensions to a GeoPackage.
+In an Extended GeoPackage, every extension SHALL be registered in a corresponding row in the `gpkg_extensions` table. An extension SHALL NOT modify the definition or semantics of existing columns. An extension MAY define additional tables or columns. An extension MAY allow new values or encodings for existing columns.
```

Comment:
One could argue that defining additional values or encodings for existing columns potentially modifies the column's definition and/or semantics.